### DISCUSSION TITLE
When will v1.0 be released? (does it still target June 2023?)

### DISCUSSION THREAD

**Discussion by woshialex** [isaac-sim/IsaacLab#85] [General]
I want to check whether the development plan is still more or less following the development roadmap? (so I could wait for it instead of developing something similar myself, thanks!)

**Comment by Mayankm96** [comment]
Hi @woshialex ,

Due to other deadlines and Kit 105 migration, the progress on releasing all the environments has slowed down. We still plan to ramp up the development in the later half of June (i.e., after the CoRL deadline) and release a majority of the environments mentioned in the paper that are possible to run with the Isaac Sim 2022.2.1 release.

If there are any particular environments that you are looking for, I can provide a more exact timeline for them.

**Reply by jmcoholich-hri** [reply]
Hi @Mayankm96,

I am interested in using the Allegro Hand environment (seen on the left at 2:58 in the video on this page https://isaac-orbit.github.io/). Are there any updates on the release of this? I would be interested in helping out as well.

Best,
Jeremiah
